db/migrations/20201028150700_podcasts.js

```diff
@@ -3,17 +3,18 @@ exports.up = function (knex) {
     return knex.schema
       .createTable('podcasts', function (table) {
         table.increments('id').primary();
-        table.string('podcast_id');
         table.string('podcast_name');
         table.string('podcast_title');
         table.string('author');
-        table.string('audience');
+        table.boolean('familiy_friendly').defaultTo(1);
```

Comment:
Do we still need this `family_friendly` boolean if we are referencing the `type` for explicit vs not explicit content?

Reply:
We don't you are right, making the change. I thought I had removed it but apparently I was wrong :P